docs/toolbox/accessibility_indicators/heatmaps/gravity.md

```diff
@@ -275,41 +275,41 @@ In order to classify the accessibility levels that were computed for each grid c
 #### Calculation travel times
 The following example illustrates how the local accessibility heatmap is computed. The travel times are calculated for each grid cell to the concerning destination on the street network.
 
-TODO: Add image of simplified hexagonal heatmap example
+![Hexagonal Grid on Heatmap](/img/toolbox/accessibility_indicators/heatmaps/gravity_based/Traveltime_grid.webp "Hexagonal Grid on Heatmap")
 
 For the hexagon shown here, the calculation yields the following results, depending on the sensitivity parameter:
 
-Uniform sensitivity parameter:
-TODO: Add image for accessibility calculation in new GOAT v2.0
+##### Uniform sensitivity parameter:
+🚨🚨 TODO: Add image for accessibility calculation in new GOAT v2.0
 
-Varying sensitivity parameter for Hypermarket:
-TODO: Add image for accessibility calculation in new GOAT v2.0
+##### Varying sensitivity parameter for Hypermarket:
+🚨🚨 TODO: Add image for accessibility calculation in new GOAT v2.0
 
 Applied in GOAT, the following differences arise:
 
 #### Calculation with uniform sensitivity parameter
 In the first example, the accessibility for grocery shops in 15 min is calculated using a uniform sensitivity parameter (β=300,000) for all shops. The result looks like this:
 
-TODO: Image for GOAT local accessibility to grocery shops
+![Local Accessibility to grocery shops](/img/toolbox/accessibility_indicators/heatmaps/gravity_based/Local_accessibility_grocery.webp "Local Accessibility to grocery shops")
```

Comment:
For now, can we hide this section which shows images of varying sensitivity values and results? As this cannot be varied in GOAT at the moment. Just maybe do a standard heatmap without mentioning sensitivity.

Reply:
Also, maybe a GIF would be nice here, but we can add this later.